packages/react/src/components/form/error/index.tsx

```diff
@@ -173,20 +174,24 @@ function ContactSupportPrompt() {
   }
 
   return (
-    <div
-      className={clsx("sid-form-error-contact-support", styles.supportPrompt)}
-      data-testid="sid-form-error-support-prompt"
-    >
-      <span>{text["error.contactSupport.prompt"]}</span>
-      <a
-        className={styles.supportCta}
-        target="_blank"
-        href={supportURL}
-        rel="noreferrer"
+    <>
+      <Divider>{text["initial.divider"]}</Divider>
```

Comment:
We shouldn't reuse the text tokens based on value. This has a different context than the `initial` state - please add another text token.

Reply:
oh sorry I missed that it's `initial`, will do!